src/gui/DatabaseWidget.cpp

```diff
@@ -825,7 +825,7 @@ void DatabaseWidget::unlockDatabase(bool accepted)
         return;
     }
 
-    Database *db = Q_NULLPTR;
+    Database* db = Q_NULLPTR;
```

Comment:
Replace it with nullptr, please. Otherwise nothing to complain.

Reply:
@phoerious fixed!